signer/src/storage/model.rs

```diff
@@ -835,7 +836,7 @@ impl From<&BitcoinBlock> for BitcoinBlockRef {
 }
 
 /// The Stacks block ID. This is different from the block header hash.
-#[derive(Debug, Copy, Clone, PartialEq, Eq, PartialOrd, Ord, Hash)]
+#[derive(Debug, Copy, Clone, PartialEq, Eq, PartialOrd, Ord, Hash, Serialize)]
```

Comment:
We should add `#[serde(transparent)]` to these.

Reply:
[a8582d9](https://github.com/stacks-network/sbtc/pull/1251/commits/a8582d963a087e7a1aa0fc58299183e077e4b195)